css/styles.css

```diff
@@ -53,83 +53,96 @@ img {
   height: 20px;
 }
 
-h1 { 
-    text-align: center;
-    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+h2 {
+  text-align: center;
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  padding-top: 20px;
 }
 
-p { 
-    font-family: Arial, Helvetica, sans-serif;
-    line-height: 3;
-    padding: 45px;
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 3;
+  padding: 45px;
 }
 
-div.centered {
-    position: absolute;
-    top: 40%;
-    left: 50%;
-    color: white;
-    transform: translate(-50%, -50%);
-    font-size: 35px;
-    font-family: Arial, 'Segoe UI';
-    font-weight: bold;
-    text-shadow: 1px 1px 2px black;
-  }
-
-
-  .container { 
-    position: relative;
-    text-align: center;
-    color: white;
-    width: 100%;
+h1 {
+  text-align: center;
+  margin: 0;
+  color: white;
+  font-size: 35px;
+  font-family: Arial, "Segoe UI";
+  font-weight: bold;
+  text-shadow: 1px 1px 2px black;
+  padding: 0 20px;
+}
+
+header {
+  min-height: 40vh;
+  background: url(./../images/playstation-background.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
I reworked the styles here so that the `header` at the top has a minimum height, a background, and display flex center. I then put an `h1` inside of it with your title. This cut down on the markup in HTML, and made it more responsive.